static/styles/compose.scss

```diff
@@ -48,11 +48,52 @@
         font-weight: 400;
         line-height: 1em;
     }
-}
 
-#compose_buttons .button.small {
-    font-size: 1em;
-    padding: 3px 10px;
+    .button.small {
+        font-size: 1em;
+        padding: 3px 10px;
+    }
+
+    .button.rounded.has-dropdown {
+        border-radius: 4px 0px 0px 4px;
+        border-right: none;
+    }
+
+    .button.dropdown {
+        border-radius: 0px 4px 4px 0px;
+        padding: 3px 3px;
+        position: relative;
+
+        .menu-open {
+            background-color: hsl(0, 0%, 30%);
+        }
+
+        .menu {
+            display: none;
+            position: absolute;
+            bottom: 0px;
+            right: 0px;
+            padding: 3px 0px;
+            margin-bottom: 24px;
```

Comment:
Note that this is hardcoding the button height to offset from the bottom, not ideal.

Reply:
"button height" being offset by the `margin-bottom: 24px` line if that's not clear